client-app/components/atoms/radio-button/radio-button.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="flex flex-row items-center space-x-2">
+    <input
+      :id="id"
+      class="appearance-none rounded-full h-6 w-6 border-2 border-gray-300 bg-white checked:border-8 checked:border-yellow-500 focus:outline-none cursor-pointer"
+      type="radio"
+      :value="value"
+      :checked="isChecked"
+      @change="$emit('update:modelValue', value)"
+    />
+    <label :for="id" class="text-base font-medium" :class="{ 'text-gray-500': !isChecked }">{{ label }}</label>
```

Comment:
Maybe it's better to wrap label in named slot to allow passing icons and other dynamic content?